swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/TemporalThresholdRule.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.swabbie.aws
+
+import com.netflix.spinnaker.swabbie.aws.exclusions.Age
+import com.netflix.spinnaker.swabbie.aws.exclusions.TemporalTagExclusionSupplier
+import com.netflix.spinnaker.swabbie.aws.model.AmazonResource
+import com.netflix.spinnaker.swabbie.model.Result
+import com.netflix.spinnaker.swabbie.model.Rule
+import com.netflix.spinnaker.swabbie.model.Summary
+import com.netflix.spinnaker.swabbie.model.WorkConfiguration
+import org.springframework.stereotype.Component
+
+/**
+ * A general rule that applies when temporal tags are found on an amazon resource
+ * When the resolved resource age is older than today, the resource will be marked for deletion if
+ * [WorkConfiguration.rule] attribues match the resource
+ */
+@Component
```

Comment:
I find this confusing at first glance. Can you give an example with a tag and the work configuration and when it would be deleted here in this javadoc?